core/src/banking_stage/consumer.rs

```diff
@@ -743,13 +743,10 @@ impl Consumer {
         let fee_payer = message.fee_payer();
         let budget_limits =
             process_compute_budget_instructions(message.program_instructions_iter())?.into();
-        let fee = bank.fee_structure().calculate_fee(
+        let fee = solana_fee::calculate_fee(
             message,
             bank.get_lamports_per_signature(),
```

Comment:
To keep inline with sdk version, the parameter should be `bank.fee_structure().lamports_per_signature`, which could be different from `bank.get_lamports_per_signature() { bank.fee_rate_governor.lamports_per_signature }`. iirc, fee_rate_governor's lamports_per_signature can change based on block utilization, but fee_structure's sig rate is static (need to verify it)

Wondering how is this not break tests?

Reply:
I don't think it actually changes, was just "future proofed" in order to allow it to change.

Reply:
but yeah I'll revert t using the fee_structure one; and will need some manual fix for all the tests that breaks.